README.md

```
@@ -16,21 +14,32 @@ If you are looking for a Laravel package designed to streamline the development
+    * Seeed Studio (XIAO 7.5" ePaper Panel)
+    * reTerminal E1001 Monochrome ePaper Display
+    * Custom ESP32 with TRMNL firmware
+    * E-Reader Devices
+      * KOReader ([trmnl-koreader](https://github.com/usetrmnl/trmnl-koreader))
+      * Kindle ([trmnl-kindle](https://github.com/usetrmnl/byos_laravel/pull/27))
+      * Nook ([trmnl-nook](https://github.com/usetrmnl/trmnl-nook))
+      * Kobo ([trmnl-kobo](https://github.com/usetrmnl/trmnl-kobo))
+    * Android Devices with [trmnl-android](https://github.com/usetrmnl/trmnl-android)
+    * Raspberry Pi (HDMI output) [trmnl-display](https://github.com/usetrmnl/trmnl-display)
 * 🔄 TRMNL API Proxy – Can act as a proxy for the native cloud service (requires TRMNL Developer Edition).
     * This enables a hybrid setup – for example, you can update your custom Train Monitor every 5 minutes in the morning, while displaying native TRMNL plugins throughout the day.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)